addons/live-featured-project/addon.json

```diff
@@ -55,33 +68,36 @@
     },
     {
       "id": "autoPlay",
-      "name": "Auto-play projects (for alternative players)",
+      "name": "Auto-play projects",
       "type": "boolean",
       "default": false,
       "if": {
         "settings": { "alternativePlayer": ["turbowarp", "forkphorus"] }
       }
     },
     {
-      "id": "forceAlternative",
-      "name": "Force using alternative player",
+      "id": "enableTWAddons",
+      "name": "Enable addons",
       "type": "boolean",
-      "default": false,
+      "default": true,
       "if": {
-        "settings": { "alternativePlayer": ["turbowarp", "forkphorus"] }
+        "settings": { "alternativePlayer": "turbowarp" }
       }
     },
     {
-      "id": "enableTWAddons",
-      "name": "Enable TurboWarp addons",
+      "id": "shareUsername",
+      "name": "Use Scratch username",
```

Comment:
Interestingly, this setting does not exist in the `TurboWarp button` addon. I wonder if we should add it.

Reply:
I thought it did, but you're right.

Reply:
Actually it always uses the Scratch username in replace player mode.

Reply:
Yeah, I wasn't clear enough. I was wondering if we should be providing a way to turn that off. Completely off-topic to this PR, though